money_bot/handlers/earn_handlers.py

```diff
@@ -0,0 +1,76 @@
+from aiogram import Dispatcher, types
+
+from money_bot.initialization import bot
+from money_bot.utils import db_utils, markups
+from money_bot.utils.states import GlobalStates
+from money_bot.utils.strings import EARN_MENU_TEXT, config
+
+
+async def entry_point(message: types.Message, new=False, last_message=None, user_id=None):
+    if user_id is None:
+        user_id = types.User.get_current().id
+    user = await db_utils.get_current_user(user_id)
+    if new or user.current_task_id == -1:
+        task = await db_utils.get_next_task(user_id)
+    else:
+        task = await db_utils.get_current_task(user_id)
+    if task is None:
+        if last_message is None:
```

Comment:
Why do you need to check if the `last_message` exist? Isn't it always exist?

Reply:
If `last_message` is None, bot create new message, so if function get `last_message`, it means that bot need to edit created message